docs/pages/playground.vue

```diff
@@ -8,22 +8,678 @@
+            :maxLines="2"
+          />
+        </template>
+        <template #footer>
+          <div>
+            <div class="footer-top-header">
+              <span>
+                <KIcon icon="createShaded" class="icon-size" />
+                Practice
+              </span>
+              <span
+                :style="{ 'background-color': $themePalette.grey.v_100,'padding': '2px' }"
+              >
+                Short Activity
+              </span>
+            </div>
+
+            <KGrid>
+              <KGridItem
+                :layout12="{ span: 8 }"
+                :layout8="{ span: 6 }"
+              >
+                <progress
+                  value="20"
+                  max="100"
+                  style="height:0.5em;width:100%"
+                ></progress>
+              </KGridItem>
+              <KGridItem
+                :layout12="{ span: 4 }"
+                :layout8="{ span: 2 }"
+              >
+                <div class="" style="float:right;">
+                  <KIcon icon="schedule" class="icon-size" />
+                  <KIcon icon="download" class="icon-size" />
+                </div>
+              </KGridItem>
+            </KGrid>
+          </div>
+        </template>
+      </KCard>
+    </ul>
+
+    <ul class="horizontal-cards-smaller">
+      <KCard
+        class="card"
+        :to="{ name: '/' }"
+        :headingLevel="2"
+        layout="horizontal"
+        thumbnailDisplay="none"
+        thumbnailAlign="right"
+        title="This is a sample resource title"
+      />
+    </ul>
+
+    <ul class="horizontal-cards-smaller">
+      <KCard
+        class="card"
+        :to="{ name: '/' }"
+        :headingLevel="1"
+        layout="horizontal"
+        thumbnailDisplay="none"
+        thumbnailAlign="left"
+        title="This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title"
+      >
+        <template #aboveTitle>
+          <KGrid>
+            <KGridItem
+              :layout12="{ span: 8 }"
+              :layout8="{ span: 6 }"
+            >
+              <span><KIcon icon="readSolid" :style="{ 'font-size': '14px' }" /></span>
+              <span
+                :style="{ 'color': $themePalette.grey.v_400,
+                          'font-size': '12px' }"
+              >Rutrum pellentesque utrum...</span>
+            </KGridItem>
+
+            <KGridItem
+              :layout12="{ span: 4 }"
+              :layout8="{ span: 2 }"
+            >
+              <div style="float:right">
+                <KIcon icon="infoOutline" class="icon-size" />
+              </div>
+            </KGridItem>
+          </KGrid>
+        </template>
+        <template #belowTitle>
```

Comment:
It seems that we are missing some padding at the bottom if we dont have footer.

![image](https://github.com/user-attachments/assets/1e53e356-75bc-4ec9-837f-b6832be2a493)

Reply:
I took care of this in https://github.com/learningequality/kolibri-design-system/pull/748